lunofono/lunofono_player
# Improve CI and upgrade dependencies

- pub: Update depedencies
- pub: Upgrade dependencies
- ci: Move Java version config to env variable
- ci: Test flutter channels instead of versions
- ci: Use a better action for the analyze step
- ci: Split tests as static and unit
- ci: Save coverage reports as artifacts
- Share tests between CI and local github hooks
- Remove automerge workflow
- Add workflow to check for WIP PRs
- ci: Test only the Flutter beta channel
- ci: Remove credentials for dependencies checkout
